scripts/clear_apps_temp_files.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# to run as cronjob every night
+
+NOW="$(date +'%Y-%m-%d %T')"
+FOLDER_PATH_ENV='APARTMENT_DATA_TRANSFER_PATH'
+FOLDER_PATH_DEFAULT='transfer_files'
+DAYS_DEFAULT=7
+dry_run=false
+
+# Print help text and exit.
+if [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: This script deletes files in provided folder older then provided days."
+  echo "       If folder not provided default is env variable $FOLDER_PATH_ENV."
+  echo "       If folder not provided default is env variable $FOLDER_PATH_DEFAULT."
+  echo "       If days not provided default is $DAYS_DEFAULT"
+  echo "       To just print files to be deleted provide -x option"
+  echo "Examples:"
+  echo " $0 -p 'path' -d 'days'"
+  echo " $0 -x"
+  exit 1
+fi
+
+while getopts ":p:d:x" opt; do
+  case $opt in
+    p) FOLDER_PATH="$OPTARG"
+    ;;
+    d) DAYS="$OPTARG"
+    ;;
+    x) dry_run=true
+    ;;
+    \?) echo "Invalid option -$OPTARG" >&2
+    exit 0
+    ;;
+  esac
+done
+
+if [[ -z $FOLDER_PATH ]]; then
+    echo "$NOW $0 WARNING: Folder path argument not provided"
+    echo "$NOW $0 INFO: Setting folder path to env variable $FOLDER_PATH_ENV"
+    FOLDER_PATH=$(printenv FOLDER_PATH_ENV)
+    if [ -z $FOLDER_PATH ]; then
+        echo "$NOW $0 WARNING: ENV variable $FOLDER_PATH_ENV not set"
+        FOLDER_PATH=$FOLDER_PATH_DEFAULT
+        echo "$NOW $0 INFO: Setting folder path to default $FOLDER_PATH"
+    fi
+fi
+
+if [[ -z $DAYS ]]; then
+    echo "$NOW $0 WARNING: Days parameter not provided"
+    DAYS=7
+    echo "$NOW $0 INFO: Setting days to 7"
+fi
+if [ ! -d $FOLDER_PATH ]; then
+    echo "$NOW $0 WARNING: Folder $FOLDER_PATH does not exist"
+    echo "Exiting..."
+    exit 1
+fi
+if "$dry_run"; then
+    echo 'Searching...' $FOLDER_PATH
```

Comment:
Dry-run is good a idea. I prefer more that if the Dry-run has given, the script tells in the beginning that this is dry-run and then it executes the code like it should do without there has changed echo texts but of course it doesn't do any actions.